integration-tests/general/filesystem-db-enclave/build.gradle

```diff
@@ -9,7 +9,7 @@ dependencies {
     //  Using 1.18.2 version for JTS triggers a bug, we are using 1.16.0 (see versions.gradle)
     runtimeOnly "org.locationtech.jts:jts:$locationtech_jts_version"
     runtimeOnly "org.locationtech.jts:jts-core:$locationtech_jts_version"
-    runtimeOnly "org.locationtech.spatial4j:spatial4j:$locationtech_spatial4js_version"
+    runtimeOnly "org.locationtech.spatial4j:spatial4j:$locationtech_spatial4j_version"
```

Comment:
The library is called `spatial4j`, I have renamed as it is annoying when searching online